features/capture-handle.yml

```diff
@@ -1,5 +1,4 @@
-draft_date: 2024-09-09
-name: Capture Handle - Bootstrapping Collaboration when Screensharing
+name: Capture handle
```

Comment:
nit: Capitalize "Handle"

Reply:
Per https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md we use sentence case for feature names, even though this sometimes looks funny. There's some discussion of it in https://github.com/web-platform-dx/web-features/issues/548.